src/components/MatchScore/MatchScore.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
+import styles from './matchScore.module.scss';
+import { classNames } from '../../shared/utilities';
+import { MatchScoreProps } from './MatchScore.types';
+
+export const MatchScore: FC<MatchScoreProps> = ({
+    className,
+    score = 0,
+    total = 5,
+    hideLabel = false,
+}) => {
+    const absTotal = Math.abs(total);
+    const matchScoreClasses = classNames(className, styles.matchScoreContainer);
+    return (
+        <div className={matchScoreClasses}>
```

Comment:
Is the progress bar here the right role for this component? Any thoughts @dkilgore-eightfold 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role